app/src/main/java/com/standuptracker/service/PersonService.kt

```diff
@@ -8,28 +8,27 @@ import com.standuptracker.dto.Person
 import retrofit2.Call
 import retrofit2.Callback
 import retrofit2.Response
-import java.lang.Exception
 
 class PersonService {
-    fun fetchPeople() : MutableLiveData<ArrayList<Person>> {
-        var _people = MutableLiveData<ArrayList<Person>>()
+    fun fetchPeople(): MutableLiveData<ArrayList<Person>> {
+        var people = MutableLiveData<ArrayList<Person>>()
```

Comment:
The _ variable naming convention only applies to private variables at the class level. Inside of a function, all variables are the local scope so traditional camel case is the best practice

Reply:
yes, agreed.